### PR TITLE
Develop to master

### PR DESCRIPTION
**Pull request by ThrawnCA**
- Partially prepare for CKAN 2.9 and Flask
- Drop dependency on ckanext-data-qld and just use ckanext-qgov

### CODE REVIEW

File: test/features/dataset_deletion.feature [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        Then I press "save"
+        And I wait for 10 seconds
+        Then I execute the script "document.getElementById('field-image-url').value='https://example.com'"
+        Then I fill in "name" with "res1"
+        Then I fill in "description" with "description"
+        Then I press the element with xpath "//button[@value='go-metadata']"
+        And I wait for 10 seconds
+        Then I should see "Data and Resources"
+
+    Scenario: Sysadmin deletes a dataset
```
Comment (duttonw) [inline test/features/dataset_deletion.feature:23]: Scenario's may not run in order, can you add a different dataset to be created (or check created) and then do the 'delete' step

**Review by duttonw — APPROVED** [review]
still want to do manual testing in dev before this is integrated.

File: test/features/dataset_deletion.feature (hunk 1 above)
Comment (ThrawnCA) [reply test/features/dataset_deletion.feature:23]: I've not seen this fail before from running out of order, even though it's run in multiple repositories...but I'll take a look.

Should we just combine it with the previous scenario? If creating a dataset is an inherent part of testing deletion, then we may as well, it seems.

Comment (duttonw) [reply test/features/dataset_deletion.feature:23]: Id still leave in to explain where it failed. i.e duplicate most of the above test but on a different name/ref

Comment (ThrawnCA) [reply test/features/dataset_deletion.feature:23]: Behave still gives us a line-by-line identification of where it failed.

**Pull request merged**